mtools/mlaunch/mlaunch.py

```diff
@@ -714,7 +695,7 @@ def restart(self):
         # there is a very brief period in which nodes are not reachable anymore, but the
         # port is not torn down fully yet and an immediate start command would fail. This
         # very short sleep prevents that case, and it is practically not noticable by users
-        time.sleep(0.1)
+        time.sleep(1)
```

Comment:
Why the increase from 0.1 to 1 second .. is this just an arbitrary "wait long enough"? Is there a better way to check the processes are shutdown (eg. pid still active?)